README.rst

```diff
@@ -74,14 +74,16 @@ More Information
 
 - `Ray paper`_
 - `Ray HotOS paper`_
+- `Ray Architecture v1 whitepaper`_
 
 .. _`Ray AIR`: https://docs.ray.io/en/latest/ray-air/getting-started.html
 .. _`Ray Core`: https://docs.ray.io/en/latest/ray-core/walkthrough.html
 .. _`Tasks`: https://docs.ray.io/en/latest/ray-core/tasks.html
 .. _`Actors`: https://docs.ray.io/en/latest/ray-core/actors.html
 .. _`Objects`: https://docs.ray.io/en/latest/ray-core/objects.html
 .. _`Documentation`: http://docs.ray.io/en/latest/index.html
-.. _`Ray Architecture whitepaper`: https://docs.google.com/document/d/1lAy0Owi-vPz2jEqBSaHNQcy2IBSDEHyXNOQZlGuj93c/preview
+.. _`Ray Architecture v1 whitepaper`: https://docs.google.com/document/d/1lAy0Owi-vPz2jEqBSaHNQcy2IBSDEHyXNOQZlGuj93c/preview
+.. _`Ray Architecture whitepaper`: https://docs.google.com/document/d/1tBw9A4j62ruI5omIJbMxly-la5w4q_TjyJgJL_jN2fI/preview
 .. _`Ray AIR Technical whitepaper`: https://docs.google.com/document/d/1bYL-638GN6EeJ45dPuLiPImA8msojEDDKiBx3YzB4_s/preview
```

Comment:
Should we just link to the v2 one only? We can in the v1 doc also have a forwarding link to the v2 paper.

Reply:
Probably it's okay to have both? The v1 is listed under "old" links. But good point, I'll update both papers to link to each other.